build.sbt

```diff
@@ -74,7 +74,7 @@ lazy val hat = project
   .settings(
     // Use the alternative "Ash" script for running compiled project form inside Alpine-derived container
     // as Bash is incompatible with Alpine
-    javaOptions in Universal ++= Seq("-Dapplication.mode=DEV", "-Denv=stage", "-Dhttp.port=9000", "-Dhttps.port=9001", "-Dpidfile.path=/dev/null", "-Dplay.server.pidfile.path=/dev/null", "-Dconfig.resource=docker.conf"),
+    javaOptions in Universal ++= Seq(""),
```

Comment:
Should we keep docker-specific `"-Dpidfile.path=/dev/null", "-Dplay.server.pidfile.path=/dev/null"` options in this list as they are being loaded only for container builds?

Reply:
That seems reasonable.